protobuf/module_msg.proto

```diff
@@ -301,6 +301,16 @@ message WildcardMatchCommandSetDefaultGateArg {
 }
 
 /**
+ * The module WildcardMatch has a function `get_rules()` which
+ * returns the current set of running rules.
+ */
+message WildcardMatchCommandGetRulesResponse {
+  uint64 default_gate = 1; // The default gate
+  WildcardMatchArg fields = 2; // The fields provided in initializing WCM
+  repeated WildcardMatchCommandAddArg rules = 3; //All rules provided via calls to `WilcardMatch.add(...)`
```

Comment:
Doing so would change the WildcardMatch constructor take in a non-standard parameter ("Rule" rather than "AddArg") which I thought would look weird and confusing for someone just trying to build a WildcardMatch. Are we okay with that?